keyboards/ps2avrGB/matrix.c

```diff
@@ -93,6 +93,8 @@ uint8_t matrix_scan(void) {
         }
     }
 
+    matrix_scan_user();
```

Comment:
This should actually be calling `matrix_scan_quantum()` - did you see this anywhere else? There might be some bad examples of this :)